lib/backbone-provider.js

```diff
@@ -1,25 +1,18 @@
-const { Component, Children } = require('react');
+const React = require('react');
+const { Component, Children, createElement } = React;
```

Comment:
You know what, since I'm using `createElement` I guess I didn't need to do this. I made this change initially when I was thinking I'd use JSX, and `React` needed to be in scope.

Reply:
Done in ad0b5a4

Reply:
Also we both totally meant to say destructure, not dereference